RuslanImankulov/homework1.js

```diff
@@ -0,0 +1,61 @@
+//Func to check palindrome
+const isPalindrome = (str) => {
+    let i = 0;
+    let j = str.length -1;
+    for (let z=1; z <= str.length/2; ++z) {
+        if (str[i] !== str[j]) {
+            return false
+        };
+        ++i;
+        --j;
+    }
+    return true
+  }
+  
+//Func to check palindrome v1
+  const isPalindrome_v1 = (str) => {
+    if (str.length === 1) {
+      return true
+    } else{
+      const firstPart = str.slice(0, Math.floor(str.length/2))
+      const secondPart = str.slice(-Math.floor(str.length/2))
+      if (firstPart === secondPart.split('').reverse().join('')){
+          return true
+      }
+    }
+    return false
+  }
+
+// Func to sort arrays (not best, but i tried)
+const quickSort = (data, start=0, end=data.length -1) => {
```

Comment:
Such an advice: it's easier to read code with well formatting. So no `start=0` but `start = 0` and no `length -1` but `length - 1`. Any IDE can format the code automatically I suppose.